### ISSUE TITLE
Vertical-Offset not respected in export image

### ISSUE THREAD

**Issue by RFCreate** [JanDeDobbeleer/oh-my-posh#2476]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

The problem is that my prompt is render this way:
![bash omp](https://user-images.githubusercontent.com/107062289/177005858-7b498fce-313c-4dab-94b7-60fb729f3366.jpg)

So the image should look like this because I have `"vertical_offset": -1` in my config :
![omp gist image](https://gist.githubusercontent.com/shyguyCreate/387ff25579b25bff63a6bc1a7635be27/raw/2df499d1f7441254ba5456396692a216855fa12c/ohmyposhCustome.png)

But instead it looks like this, btw I use ` --rprompt-offset 40`: (difference in color no matter now)
![ohmyposhCustome](https://user-images.githubusercontent.com/107062289/177005986-7d3d2c4f-6d19-4cce-8eb3-618c8fded7a5.png)

I understand that considering "vertical-offset" in the first prompt will be for nothing, but in the next blocks they change the look of the final prompt. I even tested using a positive number instead of a negative, (meaning `"vertical-offset": 1`), and not even then it adds a extra line in the image. 

### Theme

https://gist.githubusercontent.com/shyguyCreate/387ff25579b25bff63a6bc1a7635be27/raw/2df499d1f7441254ba5456396692a216855fa12c/ohmyposhCustome.omp.json

In summary, my config has this structure:

```
"blocks": [
    {
      "alignment": "right",
      "segments": [ ...
      ],
      "type": "prompt"
    },
    {
      "alignment": "left",
      "newline": true,
      "vertical_offset": -1,
      "segments": [ ...
      ],
      "type": "prompt"
    },
    {
      "alignment": "left",
      "newline": true,
      "segments": [ ...
      ],
      "type": "prompt"
    }
  ]
```

### Log output

```Shell
No debug flag for `oh-my-posh config export image`
```

**Comment by JanDeDobbeleer** [comment]
@shyguyCreate correct. As that's a hack from the past I'm also not going to do that. In fact, I'm going to remove that from the docs. It's no longer needed now we have `rprompt` support for most shells.

**Comment by RFCreate** [comment]
> In fact, I'm going to remove that from the docs. It's no longer needed now we have `rprompt` support for most shells.

@JanDeDobbeleer  Sorry, but remove what from the docs? what is no longer needed?

Also, PR about the use of offset in the image.
